test/media-query-provider.js

```diff
@@ -131,4 +131,29 @@ describe('<MediaQueryProvider />', () => {
       }).to.not.throw();
     });
   });
+
+  if (React.Fragment) {
```

Comment:
this package is always going to have the same react version number which is a problem for this test.

In any case it seems fragments are not supported yet in enzyme: 
https://github.com/airbnb/enzyme/issues/1213

Reply:
Yeah, I realized that when I ran the tests. It still is a good test for the behavior in the version being used, IMO. I can remove it if you want, though.